Source/UI/About/IdentityView.swift

```diff
@@ -10,10 +10,30 @@ import CrashReporting
 import Logger
 import SwiftUI
 
+class IdentityViewBuilder {
```

Comment:
I think this doesn't need to be a reference type as it doesn't hold any state. Can we make it an `enum` and then `func build` could be static?

Reply:
👌